type script errors while build

#### Category
[ ] Enhancement

[ x] Bug

[ ] Question
I'm trying to use sp-pnp-js library , and getting build errors.
#### Version

Please specify what version of the library you are using: [ sp-pnp-js@3.0.5     ]

#### Expected / Desired Behavior / Question
Build should succeed 

#### Observed Behavior
The build doesn't work with sp-pnp-js library in the project.

Error - typescript - node_modules\sp-pnp-js\lib\sharepoint\clientsidepages.d.ts(60,33): error TS1005: ',' expected.
Error - typescript - node_modules\sp-pnp-js\lib\sharepoint\clientsidepages.d.ts(194,20): error TS1005: ',' expected.
Error - typescript - node_modules\sp-pnp-js\lib\sharepoint\clientsidepages.d.ts(195,20): error TS1005: ',' expected.
Error - typescript - node_modules\sp-pnp-js\lib\sharepoint\clientsidepages.d.ts(60,35): error TS2368: Type parameter name cannot be 'any'
Error - typescript - node_modules\sp-pnp-js\lib\sharepoint\clientsidepages.d.ts(194,22): error TS2368: Type parameter name cannot be 'any'
Error - typescript - node_modules\sp-pnp-js\lib\sharepoint\clientsidepages.d.ts(195,22): error TS2368: Type parameter name cannot be 'any'
Error - 'typescript' sub task errored after 2.96 s
 "TypeScript error(s) occurred."


#### Steps to Reproduce
install sp-pnp-js library in your project
and type gulp serve to build


Thanks!

Same Issue:
sp-pnp-js@3.0.6

reproduce it with:
yo @microsoft/sharepoint
npm install sp-pnp-js --save
gulp serve

Gulp Console shows:
 [typescript] TypeScript version: 2.2.2
[19:55:09] Finished subtask 'copy-static-assets' after 1.07 s
[19:55:10] Error - typescript - node_modules\sp-pnp-js\lib\sharepoint\clientsidepages.d.ts(61,33): error TS1005: ',' expected.
[19:55:10] Error - typescript - node_modules\sp-pnp-js\lib\sharepoint\clientsidepages.d.ts(104,44): error TS1005: ',' expected.
[19:55:10] Error - typescript - node_modules\sp-pnp-js\lib\sharepoint\clientsidepages.d.ts(110,40): error TS1005: ',' expected.
[19:55:10] Error - typescript - node_modules\sp-pnp-js\lib\sharepoint\clientsidepages.d.ts(208,20): error TS1005: ',' expected.
[19:55:10] Error - typescript - node_modules\sp-pnp-js\lib\sharepoint\clientsidepages.d.ts(209,20): error TS1005: ',' expected.
  
What do i have to do?

--- a/src/sharepoint/clientsidepages.ts
+++ b/src/sharepoint/clientsidepages.ts
@@ -253,7 +253,7 @@ export class ClientSidePage extends File {
      * 
      * @param escapedString 
      */
-    public static escapedStringToJson<T = any>(escapedString: string): T {
+    public static escapedStringToJson<T>(escapedString: string): T {
 
         return JSON.parse(escapedString
             .replace(/&quot;/g, `"`)
@@ -695,7 +695,7 @@ export class ClientSideText extends CanvasControl {
 
         super.fromHtml(html);
 
-        const match = /<div[^>]*data-sp-rte[^>]*>(.*?)<\/div>/i.exec(html);
+        const match = /<div[^>]*data-sp-rte[^>]*>(.*?)<\/div>$/i.exec(html);
 
         this.text = match.length > 1 ? match[1] : "";
     }
@@ -726,12 +726,12 @@ export class ClientSideWebpart extends CanvasControl {
         this.propertieJson = this.parseJsonProperties(manifest.preconfiguredEntries[0].properties);
     }
 
-    public setProperties<T = any>(properties: T): this {
+    public setProperties<T>(properties: T): this {
         this.propertieJson = properties;
         return this;
     }
 
-    public getProperties<T = any>(): T {
+    public getProperties<T>(): T {
         return <T>this.propertieJson;
     }
 

